site/sfguides/src/vhol_iceberg_snowflake_open_catalog/vhol_iceberg_snowflake_open_catalog.md

```diff
@@ -0,0 +1,575 @@
+
+![sts endpoints page](assets/img03.PNG)
+
+- In the navigation pane on the left under “Access management”, select “Policies”
+- Select “Create Policy” in the top right
+- On the next screen:
+    For Policy editor, select JSON.
+    Replace the pre-populated JSON contents of the Policy Editor with the following JSON. Note don’t click save after pasting yet.
+
+
+```json
+{
+      "Version": "2012-10-17",
+      "Statement": [
+            {
+               "Effect": "Allow",
+               "Action": [
+                  "s3:PutObject",
+                  "s3:GetObject",
+                  "s3:GetObjectVersion",
+                  "s3:DeleteObject",
+                  "s3:DeleteObjectVersion"
+               ],
+               "Resource": "arn:aws:s3:::<my_bucket>/*"
+            },
+            {
+               "Effect": "Allow",
+               "Action": [
+                  "s3:ListBucket",
+                  "s3:GetBucketLocation"
+               ],
+               "Resource": "arn:aws:s3:::<my_bucket>",
+               "Condition": {
+                  "StringLike": {
+                        "s3:prefix": [
+                           "*"
+                        ]
+                  }
+               }
+            }
+      ]
+   }
+```
+
+    Before clicking save, replace “<my_bucket>” with the name of the bucket you specified in step 2.e.i (i.e., deconnect-hol-demo-<name>) in both places. Note that the value you replace with this should just be the name of the bucket (“deconnect-hol-demo-<name>”) and not include the protocol (i.e., “s3://”)
+    In the bottom right, click “Next”
+    For “Policy name”, enter “deconnect-hol-demo-policy”
+    Scroll down, and in the bottom right click “Create policy”
+
+<!-- ------------------------ -->
+
+## Create an IAM role to grant privileges on your S3 bucket
+
+- In the navigation pane on the left under “Access management”, select “Roles”
+- Select “Create role”
+- For “Trusted entity type”, select “AWS account”
+- Under “An AWS account”:
+    Select the radio button for “This account (<account_id>)”
+    Check the box that says “Require external ID”
+    In the text box that shows up after you check that box, enter “deconnect_hol_demo”
+    Click “Next”
+- On the “Add permissions” screen, search for and check the box next to the name of the policy created in step 3.f.v (“deconnect-hol-demo-policy”)
+- Click “Next”
+- For “Role name”, enter “deconnect-hol-demo-role”
+- Scroll down and click “Create role”
+- On the next screen, the “Roles” overview page, click on the name of the role you just created (“deconnect-hol-demo-role”)
+- In the “Summary” box, under “ARN”, click the copy icon to copy the role’s ARN to your clipboard. Paste this value in the HOL spreadsheet
+
+<!-- ------------------------ -->
+
+## Creating a Catalog in Snowflake Open Catalog
```

Comment:
@vinodhini-sd Shorten and title case the step label so the left-nav looks cleaner